exoctk/contam_visibility/resolve.py

```diff
@@ -2,7 +2,7 @@
 
 
 def resolve_target(targetName):
-    data = get_target_data(targetName)
+    data = get_target_data(targetName)[0]

```

Comment:
Not sure how this function even worked before because `get_target_data` returns a tuple and not a dictionary!